scripts/asm-installer/install_asm

```diff
@@ -642,6 +643,14 @@ needs_asm() {
   fi
 }
 
+add_trust_domain_alias() {
+  if [[ "${TRUST_DOMAIN_ALIASES}" == *${1}* ]]; then
+    echo "${1} already present in trust domain alias list"
```

Comment:
We shouldn't echo anything. If there are duplicates then it's our fault, not whoever is running the script as the end user.

Reply:
done